data/sources/category-config.ts

```diff
@@ -903,6 +925,10 @@ export const matchTable: {
       'heroic Public Event',
     ],
   },
+  {
+    sourceName: 'pinnacleops',
+    desc: ['Source: Pinnacle Ops'],
+  },
```

Comment:
open to better name for this source

Reply:
Since they use this name in-game, no reason to diverge from that at the moment. If the community comes up with a colloquial name for it can consider changing at that point.